Build fails after recent proc-macro2 dependency update

Running the recommended `cargo install` command currently fails with:
```
error[E0658]: use of unstable library feature 'proc_macro_is_available'
  --> /home/asent/.cargo/registry/src/github.com-1ecc6299db9ec823/proc-macro2-1.0.33/src/detection.rs:28:21
   |
28 |     let available = proc_macro::is_available();
   |                     ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #71436 <https://github.com/rust-lang/rust/issues/71436> for more information
   = help: add `#![feature(proc_macro_is_available)]` to the crate attributes to enable
```

This broke as of proc_macro2's v1.0.33 release, which fixed a version-detection bug in that crate.

The new behavior: it uses `proc_macro::is_available` in rust >=1.57, as this feature stabilized in this version.

This crate's pinned nightly is version 1.57, but does not yet have a stable `is_available`, causing the error.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,6 +21,7 @@ log = "0.4"
 semver = "1.0" # Keep in sync with version pulled by Cargo
 serde = { version = "1.0.130", features = ["derive"] }
 serde_json = "1.0.72"
+proc-macro2 = "=1.0.32"
 
 [dev-dependencies]
 quickcheck = { version = "1.0", default-features = false }

